src/components/Reports/GoalCalculator/GoalCard/GoalCard.test.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { AdapterLuxon } from '@mui/x-date-pickers/AdapterLuxon';
+import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
+import { render } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { DateTime } from 'luxon';
+import { GoalCard } from './GoalCard';
+
+const TestComponent: React.FC<React.ComponentProps<typeof GoalCard>> = (
+  props,
+) => (
+  <LocalizationProvider dateAdapter={AdapterLuxon}>
```

Comment:
We don't need `LocalizationProvider`. It's usually only needed when testing components that render date pickers.